salt/dex/dex.yaml

```diff
@@ -69,13 +69,30 @@ data:
       skipApprovalScreen: true
 
     staticClients:
+    - id: kubernetes
+      redirectURIs:
+      - 'urn:ietf:wg:oauth:2.0:oob'
+      name: "Velum"
```

Comment:
I guess we want a different name here, only for descriptive purposes?

Reply:
Whoops - Nice catch

Reply:
Fixed